src/killEvent.js

```diff
@@ -1,16 +1,12 @@
 function killEvent(target, name, descriptor) {
-  let oldHandler = target;
 
-  function eventCanceler(...args) {
-    args[0].stopPropagation();
-    args[0].preventDefault();
+  function eventCanceler(event) {
+    event.stopPropagation();
+    event.preventDefault();
 
-    oldHandler.apply(this, args);
+    return target.apply(this, arguments);
   }
 
-  if (typeof target === 'function') { return eventCanceler; }
```

Comment:
Sorry @darrenscerri the documentation on killEvent was not complete, I've updated it: https://croudcare.github.io/relpers/v1.1/ the killEvent was originally meant to be used has a normal function, removing this will break some people's code.

The v1.1 version as specs covering 100% of the code, check them out to see it your changes break anything.